src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -13,31 +12,27 @@
 public class OrderController {
     @Autowired
     private OrderRepository orderRepository;
-    @Autowired
-    private OrderMapper orderMapper;
 
     @GetMapping("/getOrders")
-    private List<OrderDto> getOrders() {
-        return orderMapper.orderListToOrderDtoList(orderRepository.findAll());
+    private List<Order> getOrders() {
```

Comment:
Nie łapię, teraz chcemy aby endpointy zwracały ob domenowe?

Reply:
Cytuję @maciek-prokopiuk:
Abstrahując od Twojego PR to nie widzę potrzebny posiadania Item oraz ItemDto, a także Order i OrderDto. Ogólnie Dto tworzy się jeśli potrzebujecie stworzyć "widok" na obiekt domenowy i ograniczyć go tylko do wybranych encji. W tym przypadku te klasy są 1:1. Wywaliłbym więc Dto i używał wszędzie klas modelowych - wtedy assemblery też idą do piachu ;) 

https://github.com/pplutap/JDP-1903-04/pull/21

OrderMapper oraz ItemMapper renerowały circular dependency problem.